flower/utils/broker.py

```diff
@@ -203,6 +246,8 @@ def __new__(cls, broker_url, *args, **kwargs):
             return RedisSsl(broker_url, *args, **kwargs)
         elif scheme == 'redis+socket':
             return RedisSocket(broker_url, *args, **kwargs)
+        elif scheme == 'sentinel':
```

Comment:
How about using `redis+sentinel` as a protocol name?

Reply:
I've decided to use that protocol name for consistency with Celery docs:
https://docs.celeryproject.org/en/latest/getting-started/brokers/redis.html#configuration